hack/lib/serverless.bash

```diff
@@ -43,8 +43,13 @@ function remove_installplan {
 
 function install_serverless_latest {
   deploy_serverless_operator_latest || return $?
-  deploy_knativeserving_cr || return $?
-  deploy_knativeeventing_cr || return $?
+
+  if [[ $INSTALL_SERVING == "true" ]]; then
+    deploy_knativeserving_cr || return $?
+  fi
+  if [[ $INSTALL_EVENTING == "true" ]]; then
+    deploy_knativeeventing_cr || return $?
+  fi
```

Comment:
These will currently fail via CI I believe. We'll likely have to add the vars to `vars.bash` and default both of them to true, to retain existing behavior and to not break CI.

Reply:
ah, check - tanks